install-and-configure/install/cloud-integration/oracle/usage-api-cloud-cost-integration.md

```diff
@@ -0,0 +1,125 @@
+# Oracle Usage API Cloud Cost Integration
+
+This document describes how to integrate Kubecost with the Oracle Usage API to build Cloud Costs for your tenancy. This will largely be done via the OCI CLI which you will need admin privileges on.
+
+### Before Starting
+
+Ensure that you have the oracle cli installed and configured for your account. Documentation on this can be found (here)[https://docs.oracle.com/en-us/iaas/Content/API/SDKDocs/cliinstall.htm].
+
+You will also need the tenancy id and region of the tenancy that your clusters are running in. This can be found in the Oracle Cloud console by selecting "Tenancy: <TENANCY-NAME>" from the profile drop down in the top right corner. Under "Tenancy information" the tenancy id is labeled "OCID" and the region is marked "Home region". The region name listed here will need to be changed to a region identify found (here)[https://docs.oracle.com/en-us/iaas/Content/General/Concepts/regions.htm] 
```

Comment:
```suggestion
You will also need the tenancy ID and region of the tenancy in which your clusters are running. This can be found in the Oracle Cloud console by selecting "Tenancy: <TENANCY-NAME>" from the profile drop down in the top right corner. Under "Tenancy information" the tenancy ID is labeled "OCID" and the region is marked "Home region". The region name listed here will need to be changed to a region identify found [here](https://docs.oracle.com/en-us/iaas/Content/General/Concepts/regions.htm).
```